assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
+        if ('localStorage' in window && window['localStorage'] !== null) {
+            localStorage.removeItem('splitSizes');
+        }
+    }
+
+    window.addEventListener('beforeunload', function (event) {
```

Comment:
@jdaguil `sessionStorage` seems to work nicely, no errors, clears on its own -- I've modified the code to move from `localStorage` to `sessionStorage`.